applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -19,6 +19,7 @@ The *Developer* perspective in the web console provides you the following option
 ** *Database*: Select the required database service and add it to your application.
 ** *Operator Backed*: Select and deploy the required Operator-managed service.
 ** *Helm chart*: Select the required Helm chart to simplify deployment of applications and services.
+** *Devfile*: Select a Devfile from Devfile registry to describe and define a development environment.
```

Comment:
Select a Devfile from the Devfile registry...

What exactly do you mean by "describe and define"? Would it make more sense to use "declaratively define"?